app/models/concerns/foreman_openscap/compliance_status_scoped_search.rb

```diff
@@ -9,9 +9,9 @@ def policy_search(search_alias)
       end
 
       def search_by_policy_name(_key, _operator, policy_name)
-        query = PolicyArfReport.of_policy(Policy.find_by(:name => policy_name))
+        scope = PolicyArfReport.of_policy(Policy.find_by(:name => policy_name))
                                .select(PolicyArfReport.arel_table[:arf_report_id]).to_sql
```

Comment:
No, I already did that and then wondered why my tests fail. To remove it, it would have to be `ArfReport.joins(:policy_arf_report).rest_of_query_here.select(ArfRepoort.arel_table[:id]).to_sql`, but we can get the report id from `PolicyArfReport`, which saves us one join.

Reply:
oh I missed we have `query_conditions_from_scope` and `query_conditions`. It's just confusing that it was renamed to `scope` while in fact it's already SQL query. But I can live with that :-)